core/io/packet_peer_udp.cpp

```diff
@@ -210,6 +211,7 @@ Error PacketPeerUDP::connect_to_host(const IP_Address &p_host, int p_port) {
 	ERR_FAIL_COND_V(udp_server, ERR_LOCKED);
 	ERR_FAIL_COND_V(!_sock.is_valid(), ERR_UNAVAILABLE);
 	ERR_FAIL_COND_V(!p_host.is_valid(), ERR_INVALID_PARAMETER);
+	ERR_FAIL_COND_V_MSG(p_port < 1 || p_port > 65535, ERR_INVALID_PARAMETER, "The server port number must be between 0 and 65535 (inclusive).");
```

Comment:
```suggestion
	ERR_FAIL_COND_V_MSG(p_port < 1 || p_port > 65535, ERR_INVALID_PARAMETER, "The server port number must be between 1 and 65535 (inclusive).");
```

Reply:
`0` is correct here,

Reply:
No, `1` is correct here, got tricked by GH partial diff. I'll fix the message.